tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -567,6 +567,10 @@ private static void InitLiveDebugger(Tracer tracer)
 
                             await InitializeLiveDebugger(liveDebugger).ConfigureAwait(false);
                         }
+                        else
+                        {
+                            Log.Warning("Live Debugger could not be initialized. Reason: Discovery service.");
+                        }
```

Comment:
nit: that's not what this means AFAICT - isDiscoverySuccessful will always be true _unless_ the app shuts down before initialization has finished, and I don't think we should log in that case

```suggestion
```

Reply:
Fixed in 6853dece4bb3cc91a684157e03ae7d3a18a3de0a